src/sketch.js

```diff
@@ -17,5 +59,50 @@ function setup()
  */
 function draw() 
 {
+    // set background color
+    background(...BACKGROUND_COLOR);
+
+    // rotate 60 degrees
+    rotateX(PI / 3);
+
+    // adjust sketch centering
+    translate(-FIELD_WIDTH / 2, -FIELD_HEIGHT / 2);
+
+    // set mesh color
+    strokeWeight(0);
+    fill(...TERRAIN_COLOR);
+
+    // increase noise parameter
+    perlin -= 0.18;
+
+    // update terrain heights
+    coordinates.y = perlin;
+    for(let y = 0; y < ROWS; y++)
+    {
+        coordinates.x = 0;
+        for(let x = 0; x < COLUMNS; x++)
+        {
+            terrain[x][y] = map(noise(coordinates.x, coordinates.y), 0, 1, -100, 100);
+            coordinates.x += 0.2
+        }
+        coordinates.y += 0.2
+    }
```

Comment:
Can we keep consistency on how to iterate the matrix? Here you iterate row by row, in contrast during assignment you iterate through columns. Maybe also rename the `x` and `y` might make it easier to know how the terrain matrix looks like.

Reply:
No, we can't. There's definitely something strange going on but it's what is creating the cool cloud effects, I'll study this with more depth later and realize what I really did, for now I think we should merge and move on.

Reply:
`x` and `y` are relative to the noise space coordinates, I think it's consistent as it is.